subprocess.h

```diff
@@ -342,7 +360,7 @@ __declspec(dllimport) unsigned long __stdcall WaitForMultipleObjects(
 __declspec(dllimport) int __stdcall GetOverlappedResult(void *, LPOVERLAPPED,
                                                         unsigned long *, int);
 
-#if defined(_DLL) && (_DLL == 1)
+#if defined(_DLL)
```

Comment:
Why this change?

Reply:
MSVC either defines `_DLL=1` or doesn't:
https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170

MinGW doesn't support static linking (as far as I can tell), so it behaves always in DLL mode, but unfortunately it defines `_DLL` as a macro without any value.

So to the best of my understanding, the above change should not break MSVC and include MinGW in the logic.

Reply:
Great explanation, thanks!